TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/datahandlers/http/NettyDecodedHttpRequestHandler.java

```diff
@@ -59,6 +60,11 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) {
             }
         } else if (msg instanceof HttpContent) {
             ctx.fireChannelRead(msg);
+        } else {
+            // ByteBufs shouldn't come through, but in case there's a regression in
+            // RequestPipelineOrchestrator.removeThisAndPreviousHandlers to remove the handlers
+            // in order rather in reverse order
```

Comment:
Should we add logging here to catch this unexpected case

Reply:
I wondered about that too, but at the end of the day, it's pretty typical that netty handlers just forward unknown messages to the next handler.  This handler would still behave correctly, right now the rest of the pipeline would too.  There's likely almost no performance impact.  It's more a comment to give a future developer a bit of insight. 
Honestly though, I'm not sure that I would have cared - I probably wrote it mostly because I had spent so long trying to figure out the remaining bytes problem that this is a part of.